SPECS/autoconf/autoconf.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
- "Signatures": {
-  "autoconf-2.71.tar.xz": "f14c83cfebcc9427f2c3cea7258bd90df972d92eb26752da4ddad81c87a0faa4"
- }
+  "Signatures": {
```

Comment:
nit: whitespace added

Reply:
This extra space was to improve the JSON file's indentation. I thought that aligning to a standard indentation of 2 spaces, as commonly for JSON formatting.